res/controllers/Numark_DJ2GO2_Touch_scripts.js

```diff
@@ -0,0 +1,214 @@
+// Declare the variable for your controller and assign it to an empty object
+var DJ2GO2Touch = {};
+
+// Mixxx calls this function on startup or when the controller
+// is enabled in the Mixxx Preferences
+DJ2GO2Touch.init = function() {
+    // create an instance of your custom Deck object for each side of your controller
+    DJ2GO2Touch.leftDeck = new DJ2GO2Touch.Deck([1], 0);
+    DJ2GO2Touch.rightDeck = new DJ2GO2Touch.Deck([2], 1);
+};
+
+DJ2GO2Touch.shutdown = function() {
+    // send whatever MIDI messages you need to turn off the lights of your controller
+    DJ2GO2Touch.leftDeck.shutdown();
+    DJ2GO2Touch.rightDeck.shutdown();
+};
+
+DJ2GO2Touch.PAD_SIZE = 4;
```

Comment:
The name is confusing, Maybe something like `numberOfPads` is better?

Reply:
good idea